target/linux/ath79/image/tiny.mk

```diff
@@ -1,5 +1,25 @@
 include ./common-buffalo.mk
 
+DEVICE_VARS += ENGENIUS_IMGNAME
+
+# This needs to make /tmp/_sys/sysupgrade.tgz an empty file prior to
+# sysupgrade, as otherwise it will implant the old configuration from
+# OEM firmware when writing rootfs from factory.bin
+define Build/engenius-tar-gz
```

Comment:
this looks identical to the one already used for the generic subtarget? If so --> `common-engenius.mk`

Reply:
ok will do